ci-operator/step-registry/vsphere/provision/haproxy/vsphere-provision-haproxy-commands.sh

```diff
@@ -8,11 +8,11 @@ set -o pipefail
 mirror_registry_url=$(< "${SHARED_DIR}"/mirror_registry_url)
 
 #Get haproxy-router image for upi disconnected installation
-target_release_image="${mirror_registry_url}/${RELEASE_IMAGE_LATEST#*/}"
+target_release_image="${mirror_registry_url}/${OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE#*/}"
```

Comment:
Suggest to add one more print line for `OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE`  for source image info to help easy debugging.